docs/sources/datasources/tempo/query-editor/index.md

```diff
@@ -19,13 +19,26 @@ The Tempo data source's query editor helps you query and display traces from Tem
 This topic explains configuration and queries specific to the Tempo data source.
 For general documentation on querying data sources in Grafana, see [Query and transform data]({{< relref "../../../panels-visualizations/query-transform-data" >}}).
 
-## Query Tempo search
+## Query by search
 
-Tempo search is an experimental feature behind a feature toggle.
 Use this to search for traces by service name, span name, duration range, or process-level attributes that are included in your application's instrumentation, such as HTTP status code and customer ID.
 
 To configure Tempo and the Tempo data source for search, refer to [Configure the data source]({{< relref "../#configure-the-data-source" >}}).
 
+**To search for traces:**
+
+1. Select **Search** from the **Query** type selector.
+1. Fill out the search form:
+
+| Name             | Description                                                                                                                       |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| **Service Name** | Returns a list of services.                                                                                                       |
+| **Span Name**    | Returns a list of span names.                                                                                                     |
+| **Tags**         | Sets tags with values in the [logfmt](https://brandur.org/logfmt) format, such as `error=true db.statement="select * from User"`. |
+| **Min Duration** | Filters all traces with a duration higher than the set value. Possible values are `1.2s, 100ms, 500us`.                           |
+| **Max Duration** | Filters all traces with a duration lower than the set value. Possible values are `1.2s, 100ms, 500us`.                            |
```

Comment:
Made each time value formatted with `, unless these samples are done as a single example? i.e., does the user enter 1.2s, 100ms, 500us or does the user use only one instead of the series? Formatting each separately indicates that they are individual choices; formatting them as a group implies that they can be used as a group. 
```suggestion
| **Min Duration** | Filters all traces with a duration higher than the set value. Possible values are `1.2s`, `100ms`, `500us`.                           |
| **Max Duration** | Filters all traces with a duration lower than the set value. Possible values are `1.2s`, `100ms`, `500us`.                            |
```